projects/1.18/assets/dynamictrees/lang/zh_cn.json

```
@@ -0,0 +1,119 @@
+{
+  "itemGroup.dynamictrees": "有活力的树",
+
+  "entity.dynamictrees.falling_tree": "倒下的树",
+
+  "block.dynamictrees.oak_branch": "有活力的橡木",
+  "block.dynamictrees.azalea_branch": "有活力的杜鹃木",
+  "block.dynamictrees.spruce_branch": "有活力的云杉木",
+  "block.dynamictrees.birch_branch": "有活力的白桦木",
+  "block.dynamictrees.jungle_branch": "有活力的丛林木",
+  "block.dynamictrees.dark_oak_branch": "有活力的深色橡木",
+  "block.dynamictrees.acacia_branch": "有活力的金合欢木",
+  "block.dynamictrees.crimson_branch": "有活力的绯红菌柄",
+  "block.dynamictrees.warped_branch": "有活力的诡异菌柄",
+  "block.dynamictrees.rooty_water": "含水树根",
+
+  "species.dynamictrees.oak": "橡树",
+  "species.dynamictrees.azalea": "杜鹃树",
+  "species.dynamictrees.swamp_oak": "沼泽橡树",
+  "species.dynamictrees.spruce": "云杉",
+  "species.dynamictrees.mega_spruce": "大型云杉",
+  "species.dynamictrees.birch": "白桦",
+  "species.dynamictrees.tall_birch": "高白桦",
+  "species.dynamictrees.jungle": "丛林木",
+  "species.dynamictrees.mega_jungle": "大型丛林木",
+  "species.dynamictrees.dark_oak": "深色橡树",
+  "species.dynamictrees.acacia": "金合欢",
+  "species.dynamictrees.apple_oak": "苹果树",
+  "species.dynamictrees.crimson": "绯红菌树",
+  "species.dynamictrees.mega_crimson": "大型绯红菌树",
+  "species.dynamictrees.warped": "诡异菌树",
+  "species.dynamictrees.mega_warped": "大型诡异菌树",
+  "species.dynamictrees.red_mushroom": "红色蘑菇",
+  "species.dynamictrees.brown_mushroom": "棕色蘑菇",
+  "species.dynamictrees.cocoa": "可可树",
+  "species.dynamictrees.oak_undergrowth": "橡树灌木",
+  "species.dynamictrees.jungle_undergrowth": "丛林木灌木",
+
+  "item.dynamictrees.oak_seed": "橡子",
+  "item.dynamictrees.azalea_seed": "杜鹃种子",
+  "item.dynamictrees.spruce_seed": "云杉球果",
+  "item.dynamictrees.birch_seed": "白桦絮",
+  "item.dynamictrees.jungle_seed": "丛林木荚",
+  "item.dynamictrees.cocoa_seed": "可可果荚",
+  "item.dynamictrees.dark_oak_seed": "深色橡子",
+  "item.dynamictrees.acacia_seed": "金合欢种子",
+  "item.dynamictrees.apple_oak_seed": "苹果仁",
+  "item.dynamictrees.crimson_seed": "绯红菌子实体",
+  "item.dynamictrees.warped_seed": "诡异菌子实体",
```

```suggestion
  "item.dynamictrees.crimson_seed": "绯红菌盖",
  "item.dynamictrees.warped_seed": "诡异菌盖",
```
cap和sporocarp应该还是有点区别，不过看合不合适吧（）

---

彳亍，就叫“菌盖”